src/workspace/mod.rs

```diff
@@ -604,12 +604,19 @@ impl Workspace {
     best_match
   }
 
-  pub fn diagnostics(&self) -> Vec<WorkspaceDiagnostic> {
+  pub fn diagnostics(
+    &self,
+    // TODO(nayeemrmn): Enable by default and remove for Deno 2.2.0.
```

Comment:
I think we can maybe consider this as a bug fix and just land it in a patch release?